tests/embed/test_gripnet.py

```diff
@@ -35,3 +36,98 @@ def test_gripnet_encoder(pose_data):
     assert gripnet.source_graph.conv_list[1].__repr__() == "GCNEncoderLayer(16, 16)"
     assert gripnet.s2t_graph.conv.__repr__() == "GCNEncoderLayer(64, 16)"
     assert gripnet.target_graph.conv_list[0].__repr__() == "RGCNEncoderLayer(48, 16, num_relations=854)"
+
+
+# create three supervertices
+node_feat = torch.randn(4, 20)
+edge_index = torch.tensor([[0, 1, 2, 3], [1, 2, 3, 3]], dtype=torch.long)
+edge_type = torch.tensor([0, 0, 1, 1], dtype=torch.long)
+
+supervertex1 = SuperVertex("1", node_feat, edge_index)
+supervertex2 = SuperVertex("2", node_feat, edge_index, edge_type)
+supervertex3 = SuperVertex("3", node_feat, edge_index, edge_type)
+
+# determine the superedges among them
+edge_index = torch.tensor([[0, 1, 2, 3], [1, 1, 3, 3]])
+
+superedge1 = SuperEdge(supervertex1.name, supervertex3.name, edge_index)
+superedge2 = SuperEdge(supervertex2.name, supervertex3.name, edge_index)
+
+# create a supergraph
+supergraph = SuperGraph([supervertex1, supervertex2, supervertex3], [superedge1, superedge2])
```

Comment:
Using underscores to separate multiple words. Follow [this guide](https://namingconvention.org/python/) for naming convention.